wagtail_localize/segments/extract.py

```diff
@@ -123,7 +123,17 @@ def handle_related_object_block(self, related_object):
     def handle_struct_block(self, struct_block, raw_value=None):
         segments = []
 
+        override_translatable_blocks = getattr(
+            struct_block.block, "override_translatable_blocks", None
```

Comment:
nitpick: let's call this `translatable_blocks` to be more or less in line with https://www.wagtail-localize.org/how-to/field-configuration/. In the `StructBlock` case `translatable_blocks` acts like `translatable_fields`. `override_translatable_fields` is used for partial overrides so if we go with 
`override_translatable_blocks` the logic needs to change

Reply:
I was having second thoughts but did originally like "translatable_fields". Will change back.